src/components/course-enrollments/CourseEnrollments.jsx

```diff
@@ -170,9 +170,9 @@ CourseEnrollments.propTypes = {
   fetchCourseEnrollments: PropTypes.func.isRequired,
   clearCourseEnrollments: PropTypes.func.isRequired,
   courseRuns: PropTypes.shape({
-    in_progress: PropTypes.array.isRequired,
-    upcoming: PropTypes.array.isRequired,
-    completed: PropTypes.array.isRequired,
+    in_progress: PropTypes.array.isRequired, // eslint-disable-line react/forbid-prop-types
+    upcoming: PropTypes.array.isRequired, // eslint-disable-line react/forbid-prop-types
+    completed: PropTypes.array.isRequired, // eslint-disable-line react/forbid-prop-types
```

Comment:
nit: these eslint errors could likely be mitigated like this:

`in_progress: PropTypes.arrayOf(PropTypes.shape())`

Basically eslint is just complaining that a prop type of `array` is too generic; it wants to know what type of elements are in the array.